Normalize Path  doesn't appear to work with .wrap

```
actix_web::App::new()
            .wrap(actix_web::middleware::Compress::default())
            .wrap(actix_web::middleware::NormalizePath::default())
```
Results in the following output 
```
error[E0277]: the trait bound `actix_web::middleware::normalize::NormalizePath: actix_service::transform::IntoTransform<_, <impl actix_service::NewService as actix_service::NewService>::Service>` is not satisfied
  --> src/main.rs:59:14
   |
59 |             .wrap(actix_web::middleware::NormalizePath::default())
   |              ^^^^ the trait `actix_service::transform::IntoTransform<_, <impl actix_service::NewService as actix_service::NewService>::Service>` is not implemented for `actix_web::middleware::normalize::NormalizePath`
```

--- a/src/middleware/normalize.rs
+++ b/src/middleware/normalize.rs
@@ -13,14 +13,30 @@ use crate::Error;
 /// Performs following:
 ///
 /// - Merges multiple slashes into one.
+///
+/// ```rust
+/// use actix_web::{web, http, middleware, App, HttpResponse};
+///
+/// fn main() {
+///     let app = App::new()
+///         .wrap(middleware::NormalizePath)
+///         .service(
+///             web::resource("/test")
+///                 .route(web::get().to(|| HttpResponse::Ok()))
+///                 .route(web::method(http::Method::HEAD).to(|| HttpResponse::MethodNotAllowed()))
+///         );
+/// }
+/// ```
+
 pub struct NormalizePath;
 
-impl<S> Transform<S> for NormalizePath
+impl<S, B> Transform<S> for NormalizePath
 where
-    S: Service<Request = ServiceRequest, Response = ServiceResponse, Error = Error>,
+    S: Service<Request = ServiceRequest, Response = ServiceResponse<B>, Error = Error>,
+    S::Future: 'static,
 {
     type Request = ServiceRequest;
-    type Response = ServiceResponse;
+    type Response = ServiceResponse<B>;
     type Error = Error;
     type InitError = ();
     type Transform = NormalizePathNormalization<S>;
@@ -39,12 +55,13 @@ pub struct NormalizePathNormalization<S> {
     merge_slash: Regex,
 }
 
-impl<S> Service for NormalizePathNormalization<S>
+impl<S, B> Service for NormalizePathNormalization<S>
 where
-    S: Service<Request = ServiceRequest, Response = ServiceResponse, Error = Error>,
+    S: Service<Request = ServiceRequest, Response = ServiceResponse<B>, Error = Error>,
+    S::Future: 'static,
 {
     type Request = ServiceRequest;
-    type Response = ServiceResponse;
+    type Response = ServiceResponse<B>;
     type Error = Error;
     type Future = S::Future;
 

